### PR TITLE
OLS-1090 - Adding operator e2e tests pipeline

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: .tekton/integration-tests/pipelines/lightspeed-operator-e2e-test-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+              echo "---------------------------------------------"
+              dnf -y install jq python3-pip
+              export OPERATOR_SDK_VERSION=1.36.1
+              export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+              export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}
+              curl -Lo /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_linux_${ARCH}
+              chmod +x /usr/local/bin/operator-sdk
+              operator-sdk version
+              echo "---------------------------------------------"
+              oc create namespace $(params.namespace)
```
Comment (raptorsun) [inline .tekton/integration-tests/pipelines/lightspeed-operator-e2e-test-pipeline.yaml:155]: after creating the namespace, we add a label to allow in-cluster monitoring stack scrape metrics from this namespace
```
oc label namespaces openshift-lightspeed openshift.io/cluster-monitoring=true --overwrite=true
```

**Review by tisnik — COMMENTED** [review]
lgtm!

**Comment by raptorsun** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / bundle"
/override "Red Hat Konflux / ols-enterprise-contract / test-bundle"
/override "Red Hat Konflux / ols-ui-tests-416 / test-bundle"

**Pull request merged**